### PR TITLE
use ManyToManyRelatedManager.set method to save m2m fields

### LINKED ISSUE #694: import of m2m relations failing in django 2
TypeError: Direct assignment to the forward side of a many-to-many set is prohibited. Use m2m_field.set() instead.

### PR DESCRIPTION
**Pull request by trik**
fixes #694

### CODE REVIEW

**Comment by shaggyfrog** [comment]
Hi, thanks for the contribution. Please can you add a test that fails without your change, and passes with your change? Thanks!

**Comment by trik** [comment]
sure! just pushed

**Comment by bmihelac** [comment]
It looks like  travis is rejecting request to process this and few other pull requests.

https://travis-ci.org/django-import-export/django-import-export/requests

I have contacted travis support regarding this and will post here when more info is available

**Comment by bmihelac** [comment]
I just received response regarding rejected travis-ci requests and posting it below
TL;DR  @trik it looks like issue is because you never signed up for Travis CI, so I would like to kindly ask you to sign up for Travis CI if this is convenient for you. Thanks.

> I've been looking into these rejected requests and I see that the gnucoop organization that created the PR nor any of its members have been never logged into Travis CI and this is why we are currently blocking these requests. Sorry for the inconvenience.
> 
> We are looking into fixing this false positive in our abuse detection system, as we don't feel that an organization or account should have to create a Travis account when working with another project that is using Travis CI.
> 
> To sort this out in the meantime, could you please confirm if you're a member of the gnucoop organization in GitHub or ask a member of that organization to sign up at Travis CI? 
> 
> Once gnucoop is signed up, closing and reopening any of the affected PRs should make GitHub send the build event again to Travis, that should no longer block these requests.

**Comment by trik** [comment]
@bmihelac no problem, just signed in travis-ci with my github account

**Comment by bmihelac** [comment]
@trik strange, travis-ci is still rejecting. I have contacted travis-ci support for help.

**Comment by trik** [comment]
@bmihelac looking at my profile settings, i found that travis had no access to my organization (gnucoop). now it should work

**Comment by bmihelac** [comment]
it looks like it still rejecting:
https://travis-ci.org/django-import-export/django-import-export/requests

**Comment by bmihelac** [comment]
Thanks, I have tested locally and merged.
